odoo/addons/test_lint/tests/test_i18n.py

```diff
@@ -0,0 +1,128 @@
+import logging
+import re
+
+from . import lint_case
+
+from odoo import tools
+
+_logger = logging.getLogger(__name__)
+
+
+class TestI18n(lint_case.LintCase):
+    PROPS_RE = re.compile(
+        r"""
+        (
+            <[A-Z]                          # Match the opening tag of a component node. We assume that tags starting with an uppercase letter refer to a component
+                (
+                    [^>]+                   # Match anything that is not a closing tag `>`. In other words, the rest of the component name and any prop that doesn't match the heuristics that follow
+                )
+        )
+        \s
+        (?!t-)                              # exclude directives (attributes starting with t-)
+        (
+            [a-zA-Z-]+                      # Match prop name
+            =
+            "'                              # Make sure that the value is a static string literal. Only static string literal are eligible for translation. We determine that the value is a string literal if the value begins and ends with a '
+            [A-Z](\'|[^'"])*?                # Assumption: Text starting with an uppercase letter is likely supposed to be translated.
```

Comment:
"likely supposed" sounds weird to me, but maybe it's technically correct?

```suggestion
            [A-Z](\'|[^'"])*?                # Assumption: Text starting with an uppercase letter is probably supposed to be translated.
```